django/test/testcases.py

```diff
@@ -807,6 +837,23 @@ def assertXMLNotEqual(self, xml1, xml2, msg=None):
                 self.fail(self._formatMessage(msg, standardMsg))
 
 
+class _MutliDBDescriptor:
```

Comment:
It's just trivial, but `Mutli` -> `Multi`.

Reply:
I think `_DatabasesDescriptor` is a better name anyway as that's the attribute name it's a descriptor for

Reply:
The problem with `_DatabasesDescriptor` is that the `SimpleTestCase.databases` decorator is already named this way.